### DISCUSSION TITLE
publish .ipynb file with picture not working

### DISCUSSION THREAD

**Discussion by lichunyu123** [quarto-dev/quarto-cli#3782] [Q&A] [answered]
i try to publish a post which writing with jupyter file named "index.ipynb" in my blog using quarto. In the same folder i put a picture named "welcome.jpg" as a preview image. however, i render the blog, this post‘ content is correct but did not show the preview image,  then i convert the ipynb file to qwd format, and specify the preview image name, it still not working. Does quarto support post in ipynb format to display preview pictures? How can I do it?

the following shows the qwd file convered from ipynb and specify de preview image:
---
title: ' test '
subtitle: 'Lesson 3-3'
author:
  - Junchuan Yu
date: '2022-11-15'
categories:
  - Posts
  - Deep leanring
  - Teaching
  - Workshop
image: welcome.jpg
toc: true
---

**Comment by dragonstyle** [comment, marked as answer]
I can't see the formatting of your document - but that should work. Can you share the document by pasting it between sets of 3 ticks, like:

````
```
Paste document here
```
````

**Reply by JunchuanYu** [reply]
problem solved! I added some description statements at the beginning of the .ipynb  file and changed the cell format to raw.

--------
---
title: ' test'
date: '2022-11-17'
categories:
  - Posts
  - Deep leanring
  - Teaching
  - Workshop
image: welcome.png
toc: true
-----------------
